.gitignore

```diff
@@ -11,3 +11,4 @@ Cargo.lock
 *.bak
 
 *.s
+.idea/
```

Comment:
mobilecoin.git has an entry for `./idea` in its [gitignore](https://github.com/mobilecoinfoundation/mobilecoin/blob/master/.gitignore). I'd prefer to do the same here. It will save everyone time.